src/modules/image/index.ts

```diff
@@ -348,15 +348,17 @@ export class ImageModule {
   }
 
   /**
-   * Generates a random data uri containing an svg image.
+   * Generates a data uri containing an URL-encoded SVG image or a Base64-encoded SVG image.
```

Comment:
```suggestion
   * Generates a random data uri containing an URL-encoded SVG image or a Base64-encoded SVG image.
```

Although this is not random yet, that will be fixed as part of #2316.